examples/instance_template/additional_disks/main.tf

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+  project     = "${var.project_id}"
+  region      = "${var.region}"
+  version     = "~> 1.19"
+}
+
+module "instance_template" {
+  source          = "../../../modules/instance_template"
+  subnetwork      = "${var.subnetwork}"
+  service_account = "${var.service_account}"
+  name_prefix     = "additional-disks"
+
+  additional_disks = [
+    {
+      disk_size_gb = 10
+      disk_type    = "pd-standard"
+      auto_delete  = "true"
+      boot         = "false"
+    },
+    {
+      disk_size_gb = 20
```

Comment:
Would reducing these sizes save money during testing?

Reply:
Yes it would, but the difference between 10 GB and 20 GB equates to $0.40 difference per month, if the disk existed for a whole month. On the other hand, I didn't have a specific reason to request for 20 GB so I've reset all the disks to 10 GB. :)